jekyll/_cci2/runner-installation-mac.adoc

```diff
@@ -13,7 +13,7 @@ contentTags:
 :machine:
 :macos:
 
-This page describes how to install CircleCI's machine runner on macOS.
+This page describes how to install CircleCI's machine runner on macOS. Please note that this method is deprecated for cloud customers and see https://circleci.com/docs/install-machine-runner-3-on-macos/ for recommended approach. 
```

Comment:
@nathanfishcircleci are the steps on this page still correct for server? If so I can add the same wording as used in the linux page in this PR

Reply:
Still good for server.